app/src/main/java/io/github/droidkaigi/confsched/widget/NumberRatingBar.java

```diff
@@ -0,0 +1,66 @@
+package io.github.droidkaigi.confsched.widget;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+import android.widget.SeekBar;
+
+public class NumberRatingBar extends SeekBar {
+    private OnSeekBarChangeListener mUserSeekBarChangeListener;
```

Comment:
Oh, embarrassing :persevere:  In fact, I copied it from Google I/O source code. I will fix it ASAP.